test/libsolidity/lsp/include-paths/default-include.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "my-module/test.sol";
+
+contract MyContract
+{
+    function f(uint a, uint b) public pure returns (uint)
+    {
+        return MyModule.add(a, b);
+    }
+}
+// ----
+// file:///home/christianparpart/work/solidity/test/libsolidity/lsp/node_modules/my-module/test:
```

Comment:
This path looks like it's specific to your system. It shouldn't be here, should it?

Reply:
But is this something enforced by the test suite? If so, won't this fail for anyone but you? If not, why not remove it?

Reply:
@Marenz what's that line exactly there for? Maybe you can jump in?

Reply:
I am not sure how or about what you two are misunderstanding each other, but yes, this path should be relative to the root dir or rather the test dir. I believe chris pushed this only so I can investigate.